client/lib/analytics/ga.js

```diff
@@ -13,14 +13,14 @@ let initialized = false;
 
 function initialize() {
 	if ( ! initialized ) {
-		const parameters = {
-			send_page_view: false,
+		const params = {
 			...getGoogleAnalyticsDefaultConfig(),
+			send_page_view: false,
```

Comment:
Changing the order here means that `send_page_view` will always be `false`, regardless of whether it's set in `getGoogleAnalyticsDefaultConfig()`. Was that your intention with this change?